simple/msg_sockets.c

```diff
@@ -498,7 +497,8 @@ static int setup_handle(void)
 	aihints.ai_flags = AI_PASSIVE;
 	ret = getaddrinfo(opts.src_addr, opts.src_port, &aihints, &ai);
 	if (ret == EAI_SYSTEM) {
-		FT_PRINTERR("getaddrinfo", -ret);
+		fprintf("getaddrinfo for %s:%s: %s\n",
+				opts.src_addr, opts.src_port, strerror(errno));
```

Comment:
Sorry about that.  I should have tested that before submitting it.
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.